test/qunit/methods/close.js

```diff
@@ -0,0 +1,71 @@
+const { Swal } = require('../helpers')
+
+QUnit.test('close() method', (assert) => {
+  const done = assert.async()
+
+  // create a modal with an onAfterClose callback
```

Comment:
all "create a modal with an onAfterClose callback" aren't needed, please remove here and below

Reply:
Resolved by https://github.com/sweetalert2/sweetalert2/pull/1379/commits/cea3b81f0a1470bfdeb8ff18ffce92ef080b9c46